docs/content/en/docs/Advanced/livelayering.md

```diff
@@ -68,31 +68,29 @@ For example, a bundle can be defined as a naked container image containing only
 
 Consider the following Dockerfile to create an extension which adds `/usr/bin/ipfs` to the system:
 
+
+{{% alert title="Note" %}}
+Note that systemd extensions require an extension-release file, which can be used to validate different aspects of the system being run.
```

Comment:
@mudler yeah makes sense. How about this?

Reply:
perfect!